pmpro-import-users-from-csv.php

```diff
@@ -203,6 +203,14 @@ function pmproiufcsv_is_iu_post_user_import($user_id)
 	if ( ! empty( $membership_timestamp ) ) {
 		$membership_timestamp = date( 'Y-m-d', strtotime($membership_timestamp, current_time( 'timestamp' ) ) );
 	}	
+
+	if ( ! empty( $membership_discount_code ) && empty( $membership_code_id ) )
+		$membership_code_id = $wpdb->get_var("SELECT id FROM $wpdb->pmpro_discount_codes WHERE `code` = '" . esc_sql($membership_discount_code) . "' LIMIT 1");		
+	
+	if( pmpro_hasMembershipLevel( $membership_id, $user_id ) && ( !empty( $_REQUEST['supress_change_membership_hooks'] ) && $_REQUEST['supress_change_membership_hooks'] == '1' ) ){

```

Comment:
We can simplify the logic here since it will either be set as 1 or not set/empty.

```suggestion
	if( pmpro_hasMembershipLevel( $membership_id, $user_id ) && ! empty( $_REQUEST['supress_change_membership_hooks'] ) ){
```